src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorWidget.ts

```diff
@@ -496,9 +498,14 @@ export class InteractiveEditorWidget {
 		this._elements.message.style.webkitLineClamp = expansionState === ExpansionState.NOT_CROPPED ? 'none' : (expansionState === ExpansionState.EXPANDED ? MESSAGE_EXPANDED_NUMBER_LINES.toString() : MESSAGE_CROPPED_NUMBER_LINES.toString());
 	}
 
-	updateInfo(message: string): void {
+	updateInfo(message: IMarkdownString | string): void {
 		this._elements.infoLabel.classList.toggle('hidden', !message);
-		this._elements.infoLabel.innerText = message;
+		if (isMarkdownString(message)) {
```

Comment:
I don't think we want to support full markdown here (think of tables, bullet lists etc...). For simple icon/codicon rendering you can use the [renderLabelWithIcons](https://github.com/microsoft/vscode/blob/feab5843d96a5b0d7e2cd8f7a5c0a26ac1330ad4/src/vs/base/browser/ui/iconLabel/iconLabels.ts#L10)-function